cypress/integration/images.spec.js

```diff
@@ -0,0 +1,292 @@
+
+      // ASSERTS
+      cy.wait('@deleteAlbum') // Should intercept POST request
+      cy.contains(ALBUM_TITLE).should('not.exist') // Album name should not be contained in Images
+    })
+  })
+
+  describe('Create image, delete image, edit image settings, and move images in image albums', () => {
+    before(() => {
+      cy.setCookie(COOKIE_NAME, COOKIE_VALUE)
+      window.localStorage.setItem('userId', 'test')
+
+      cy.visit(`/sites/${TEST_REPO_NAME}/images`)
+      cy.contains('Create new album').click()
+      cy.get("[id='image album']").clear().type(ALBUM_TITLE)  
+      cy.get('button').contains(/^Create$/).click() 
+
+      // Assert
+      cy.contains(ALBUM_TITLE, { timeout: CUSTOM_TIMEOUT}).should('exist')
+    })
+
+    beforeEach(() => {
+      cy.setCookie(COOKIE_NAME, COOKIE_VALUE)
+      window.localStorage.setItem('userId', 'test')
+      
+      cy.visit(`/sites/${TEST_REPO_NAME}/images/${SLUGIFIED_ALBUM_TITLE}`)
+    })
+
+    it('Should be able to add image to image album', () => {
+      // Set intercept to listen for POST request on server
+      cy.intercept({ method:'POST', url: `/v1/sites/${TEST_REPO_NAME}/images` }).as('createImage')
+
+      cy.uploadMedia(IMAGE_TITLE, TEST_IMAGE_PATH)
+      // ASSERTS
+      cy.wait('@createImage') // should intercept POST request
+      cy.contains(IMAGE_TITLE, { timeout: CUSTOM_TIMEOUT}).should('exist') // image should be contained in Images
+    })
+
+    it('Should be able to edit an image in image album', () => {
+      cy.intercept({ method:'POST', url: `/v1/sites/e2e-test-repo/images/${generateImageorFilePath(SLUGIFIED_ALBUM_TITLE, IMAGE_TITLE)}/rename/${generateImageorFilePath(SLUGIFIED_ALBUM_TITLE, OTHER_IMAGE_TITLE)}` }).as('renameImage')
+
+      cy.renameMedia(IMAGE_TITLE, OTHER_IMAGE_TITLE)
+      // ASSERTS
+      cy.wait('@renameImage')
+      cy.contains(OTHER_IMAGE_TITLE) // Image should be contained in Images
+
+      cy.intercept({ method:'POST', url: `/v1/sites/e2e-test-repo/images/${generateImageorFilePath(SLUGIFIED_ALBUM_TITLE, OTHER_IMAGE_TITLE)}/rename/${generateImageorFilePath(SLUGIFIED_ALBUM_TITLE, IMAGE_TITLE)}` }).as('renameImageBack')
+
+      cy.renameMedia(OTHER_IMAGE_TITLE, IMAGE_TITLE) // Rename image to original title 
+      // ASSERTS
+      cy.wait('@renameImageBack')
+      cy.contains(IMAGE_TITLE)
+    })
+
+    it('Should be able to delete image from image album', () => {
+      // Set intercept to listen for POST request on server
+      cy.intercept({ method:'DELETE', url: `/v1/sites/${TEST_REPO_NAME}/images/${generateImageorFilePath(SLUGIFIED_ALBUM_TITLE, IMAGE_TITLE)}` }).as('deleteImage')
+
+      cy.deleteMedia(IMAGE_TITLE)
+      // ASSERTS
+      cy.wait('@deleteImage') // Should intercept DELETE request
+      cy.contains(IMAGE_TITLE).should('not.exist') //Image file name should not exist in Images
+    })
+
+    it('Should be able to move image from image album to Images', () => {
```

Comment:
As discussed, these errors seem to be associated with running Cypress with the browser open. When running the tests in headless mode, these errors do not occur. We'll ignore these errors for now, since we have also confirmed that the file moving functionality works